packages/dx-react-chart/src/plugins/axis.tsx

```diff
@@ -102,12 +116,12 @@ class RawAxis extends React.PureComponent {
                     height: (dx * height) || undefined,
                     flexGrow: dx || undefined,
                   }}
-                  ref={this.rootRef}
+                  ref={this.rootRef as any}
                 >
                   <svg
                     width={this.adjustedWidth}
                     height={this.adjustedHeight}
-                    style={SVG_STYLE}
+                    style={SVG_STYLE as any}
```

Comment:
What kind of error is shown without `as any`?

Reply:
`Type '{ position: string; left: number; top: number; overflow: string; }' is not assignable to type 'CSSProperties'.
  Types of property 'position' are incompatible.
    Type 'string' is not assignable to type '"absolute" | "-moz-initial" | "inherit" | "initial" | "revert" | "unset" | "-webkit-sticky" | "fixed" | "relative" | "static" | "sticky" | undefined'`

Reply:
SVG_STYLE should be typed as style: 
`const SVG_STYLE: React.CSSProperties - ...`